### DISCUSSION TITLE
L.A Noire

### DISCUSSION THREAD

**Discussion by mastershing18** [skyline-emu/skyline#1749] [Show and tell]
![Screenshot_2022-08-04-19-35-17-602_skyline emu](https://user-images.githubusercontent.com/87024590/182884014-4e2f03ed-dfeb-4607-b4ea-39ce67f0f5b2.jpg)
![Screenshot_2022-08-04-19-31-57-024_skyline emu](https://user-images.githubusercontent.com/87024590/182884031-3bd04da0-e806-453a-8197-b2a54d75f3f0.jpg)

My device is poco x3 pro 6/128 i can accelerate the car but low fps and graphic bug

**Comment by Crytonics** [comment]
For games go to https://github.com/skyline-emu/skyline-games-list/issues
